src/pages/account-menu/in-an-app-bar/Chip.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Chip as MuiChip, ChipProps as MuiChipProps, Typography, makeStyles, Theme, useTheme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    userMenuChip: {
+        width: '112px',
+        height: `${theme.spacing(4)}px`,
+        cursor: 'pointer',
+    },
+    chipIcon: {
+        height: `${theme.spacing(3)}px`,
+        width: `${theme.spacing(3)}px`,
+    },
+    chipLabelIcon: {
+        height: 16,
+        width: 16,
+    },
+    chipLabelContainer: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+}));
+
+type ChipProps = MuiChipProps & {
+    label: string;
+    rightIcon: React.ReactElement;
```

Comment:
is it worth trying to make this work with the `deleteIcon` prop from MUI or is that more work than it's worth since it doesn't do delete?

Reply:
If I use `deleteIcon` prop of MUI, I am not able to open userMenu Items.
![image](https://user-images.githubusercontent.com/10433274/127268345-f6adfa87-91a3-4644-9c02-b3cec3f02d9f.png).

Do I need to use `useRef` hook to manually click userMenu ?

Reply:
Let's not use the deleteIcon. It was just a suggestion to see if it would work to save some styling, but the onDelete versus onClick causes issues when using it with the UserMenu and 'onDelete' doesn't feel semantically correct for the functionality that it's actually performing.